configuration/pih/htmlforms/hiv/section-hiv-plan.xml

```diff
@@ -498,161 +502,200 @@
-                            <td class="other-pro-treat">
-                                <obs conceptId="CIEL:163526" />
+                                <label>
+                                    <uimessage code="pihcore.treatment" />
+                                </label>
                             </td>
-                            <td class="other-pro-treat">
-                                <obs conceptId="CIEL:164384" allowFutureDates="true" />
+                            <td>
+                                <label>
+                                    <uimessage code="pihcore.hiv.statusAndOrder" />
+                                </label>
                             </td>
-                            <td class="other-pro-treat">
-                                <obs conceptId="CIEL:159367" answerConceptId="CIEL:1065"
-                                     answerLabel=""/>
+                            <td>
+                                <label>
+                                    <uimessage code="pihcore.hiv.dosage" />
+                                </label>
                             </td>
-                            <td class="other-pro-treat">
-                                <small>
-                                    <obs conceptId="CIEL:6098"
-                                         answerConceptIds="CIEL:843,CIEL:1754,CIEL:102,CIEL:987,CIEL:1434,CIEL:5622" />
-                                </small>
+                            <td>
                             </td>
                         </tr>
-                    </obsgroup>
-                </table>
+
+                        <repeat>
+                            <template>
+                                <obsgroup groupingConceptId="PIH:Prescription construct">
+                                    <tr>
+                                        <td>
+                                            <!-- Medication orders -->
+                                            <obs conceptId="CIEL:1282" answerConceptId="{conceptMap}"
+                                                answerLabel="{drugname}"
+                                                toggle="{id: '{drugname}-treat', style: 'dim'}"/>
+                                        </td>
+                                        <td class="{drugname}-treat">
+                                            <obs conceptId="PIH:PROPHYLAXIS STARTED"
+                                                 answerConceptId="CIEL:1065" style="checkbox"
+                                                 labelCode="pihcore.hiv.started" answerLabel=""
+                                                 toggle="{drugname}-start-date"/>
+                                            <obs conceptId="PIH:Prophylaxis should continue"
+                                                 answerConceptId="CIEL:1065" style="checkbox"
+                                                 labelCode="pihcore.hiv.continue" answerLabel="" />
```

Comment:
Use labelCode="emr.continue"

Reply:
Please don't use `emr.` message codes; those are from the old `openmrs-module-emr` and are defunct and untranslatable. Fortunately all of them seem to have been imported into [coreapps](https://github.com/openmrs/openmrs-module-coreapps/blob/master/api/src/main/resources/messages.properties), so you can just replace `emr.` with `coreapps.` any time you see it. So here I'll use `coreapps.continue`—thanks!